### DISCUSSION TITLE
How to proxy pass RTMP stream in deno deploy?

### DISCUSSION THREAD

**Discussion by yisar** [denoland/deno#24568] [General]
I have an RTMP server, and initially, I used nginx's proxy. However, in Deno, I couldn't find a similar method. The fetch request method is invalid because it's an RTMP request instead of HTTP. Is there any functionality similar to nginx's proxy_pass?

nginx config like this:
```nginx
stream { 
    server { 
        listen 3000; 
        proxy_pass localhost:1935; 
     } 
}
```
how to implement this with deno way?

**Comment by BlackAsLight** [comment]
Not really answering your question, but why not continue to use nginx as the proxy? It would be much more efficient at the job

**Reply by yisar** [reply]
deno deploy provides SSL certificates outofbox. I want to use it to replace nginx, but only in proxy_pass.
